chapters/hpmor-chapter-013.tex

```diff
@@ -174,7 +174,7 @@ \chapter{Die falschen Fragen stellen}
 Nun, Schritt eins war sich umziehen und etwas essen. Besser in umgekehrter Reihenfolge, sein Magen fühlte sich ziemlich leer an.
 
 % He’d missed breakfast, of course, but he was Prepared for that eventuality, having visualised it in advance. Harry put his hand into his pouch and said “Snack bars”, expecting to get the box of cereal bars he’d bought before departing for Hogwarts.
-Das Frühstück hatte er natürlich verpasst, doch darauf war er vorbereitet, da er vorher bereits geahnt hatte, dass das passieren könnte. Harry griff in den Beutel, sagte „Müsliriegel“ und ging davon aus, dass eine Schachtel Müsliriegel in seiner Hand erscheinen würde.
+Das Frühstück hatte er natürlich verpasst, doch auf diese Eventualität war er Vorbereitet, hatte sie bereits vorgeahnt. Harry griff in den Beutel, sagte „Müsliriegel“ und ging davon aus, dass eine Schachtel Müsliriegel in seiner Hand erscheinen würde.
```

Comment:
"vorbereitet" klein?

Reply:
"hatte sie bereits vorgeahnt" -> in weise Vorraussicht vorbereitet?

Reply:
Großgeschrieben, weil "Prepared" im Original ebenfalls so gebaut. "In weiser Voraussicht" käme glaub ich richtig gut!